util/rom-ext-manifest-generator.py

```diff
@@ -52,12 +53,13 @@
   rom-ext-manifest-generator --output-dir:
     Directory where manifest.h will be created.
 
-  rom-ext-manifest-generator --output-headers=[all | c | rust]:
-    Type of headers to be generated:
+  rom-ext-manifest-generator --output-files=[all | c | rust]:
+    Type of files to be generated:
 
-    all  - All header files
+    all  - All files listed below
     c    - C header file
-    rust - Rust "header" file
+    rust - Rust module file
+    format - Format description file (plaintext)
```

Comment:
nit: could you align the `-` signs (pedantic I know :smiley:), makes it slightly nicer to read (I think).